environments/common/src/main/java/org/jboss/weld/environment/logging/CommonLogger.java

```diff
@@ -147,4 +147,8 @@
     @Message(id = 39, value = "Bean archive reference {0} handled by {1}", format = Format.MESSAGE_FORMAT)
     void beanArchiveReferenceHandled(Object beanArchiveRef, Object handler);
 
+    @LogMessage(level = Level.WARN)
+    @Message(id = 40, value = "Consider using 2.0.3.Final Jandex version and higher. See JANDEX-37", format = Format.MESSAGE_FORMAT)
```

Comment:
This message is nor really useful. Add some concrete info pls. The same applies to `ClassFileInfoException`...